modules/azure/aks/variables.tf

```diff
@@ -205,6 +205,47 @@ variable "log_eventhub_authorization_rule_id" {
   type        = string
 }
 
+variable "defender_enabled" {
+  description = "If Defender for Containers should be enabled"
+  type        = bool
+  default     = false
+}
+
+variable "audit_config" {
+  description = "Kubernetes audit log configuration"
+  type = object({
+    destination_type = optional(string, "StorageAccount")
+    analytics_workspace = optional(object({
+      sku_name       = optional(string, "PerGB2018")
+      daily_quota_gb = optional(number, -1)
```

Comment:
Does -1 mean infinite?

Reply:
Yes. Do we have a better number to use?

Reply:
I just mean does it also mean infinite for the API.

Reply:
It means "The workspace daily quota for ingestion in GB", so no.